t/one-core-peak.sh

```diff
@@ -250,7 +257,18 @@ show_system() {
   fi
 }
 
+get_specific_options() {
+  # Add some io_uring benchmark features based on the kernel release
+  options=""
+  # Starting Kernel 5.16, we can enable the DMA pre-mapping
```

Comment:
Never make these things dependent on kernel version, that breaks quite often as distros backport patches. I think for this particular feature, you'd just have to try, and revert to not using -D1 if it fails...

Reply:
done.